### PR TITLE
AGU journals

### PR DESCRIPTION
**Pull request by libora**
tables for AGU journals. 
Note: It is my first attempt to create this, so I hope there is not a lot of mistakes.

### CODE REVIEW

File: agu/_journals.tab [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Journal of Geophysical Research: Oceans 2169-9275 2169-9291
+Journal of Geophysical Research: Planets  2169-9097 2169-9100
+Journal of Geophysical Research: Solid Earth  2169-9313 2169-9356
+Journal of Geophysical Research: Space Physics  2169-9380 2169-9402
+Paleoceanography  0883-8305 1944-9186
+Radio Science 0048-6604 1944-799X
+Reviews of Geophysics X 1944-9208
+Space Weather 1539-4956 1542-7390
+Tectonics 0278-7407 1944-9194
+Water Resources Research  0043-1397 1944-7973
```
Comment (zuphilip) [inline agu/_journals.tab]: Are these tabs or spaces between different cells? (It should be tabs, but it looks for me like spaces...)

Comment (libora) [inline agu/_journals.tab]: I use tabs

Comment (rmzelle) [inline agu/_journals.tab]: They really look like spaces to me, too.

Comment (libora) [inline agu/_journals.tab]: Yes me too. But there are tabs. I had twice delete it and use TAB key. Do you have any tool for controlling it?

Comment (adam3smith) [inline agu/_journals.tab]: Looked at this in notepad++ -- these are definitely spaces, sorry.

Comment (rmzelle) [inline agu/_journals.tab]: @libora, which text editor or spreadsheet software are you using?

**Comment by libora** [comment]
@rmzelle I use pspad (pspad.com), but after inserting in github I manually delete all "spacec" between items and use the TAB key. I can try to do it again, if there is still the same problem, I will try to insert text without editing.

**Comment by adam3smith** [comment]
Yes, still no tabs. The file on gdrive is good, though, so the problem here is the github editor. Try uploading without changes, if that doesn't work I'll just do this manually.

**Comment by rmzelle** [comment]
I just made a pull request for @libora, where I just copied the data from the Google Docs spreadsheet and pasted it into _journals.tab using the online editor: https://github.com/libora/journals/pull/1/files

Came out as tabs for me (Firefox, OS X). @libora, if you keep having issues you can always use a desktop program like GitHub Desktop.

**Comment by libora** [comment]
@rmzelle I merged your PR just now (I am on cottage so I have only a few moments in the day when I have available PC). I do not use Github desktop because it is not working in my office (I think due proxy settings) and editing on the web interface of Github was OK till now.

**Comment by rmzelle** [comment]
@libora, thanks! Just added the styles. Just had to fix one ISSN that was missing a digit.

We often also add a brief README to mention where we got the metadata from. Did you just copy and paste all the journal titles and ISSNs, or did you find a catalog file somewhere?

**Comment by libora** [comment]
@rmzelle I copy it directly from web page of journal and control it on Ulrichsweb.

**Comment by rmzelle** [comment]
> and control it on Ulrichsweb.

Is that where you got the ISSNs from?

**Comment by libora** [comment]
Yes

**Pull request merged**